### PR TITLE
Pass test object to latency tests

### PR DESCRIPTION
**Pull request by KaptenJansson**

### CODE REVIEW

File: src/js/bandwidth_test.js [hunk 1]
```diff
@@ -227,75 +227,95 @@ VideoBandwidthTest.prototype = {
 };
 
 addExplicitTest(testSuiteName.THROUGHPUT, testCaseName.NETWORKLATENCY,
-  Call.asyncCreateTurnConfig.bind(null, wiFiPeriodicScanTest.bind(null,
-      Call.isNotHostCandidate), reportFatal));
+  function(test) {
+    var wiFiPeriodicScanTest = new WiFiPeriodicScanTest(test);
+    wiFiPeriodicScanTest.run(Call.isNotHostCandidate);
```
Comment (andresusanopinto) [inline src/js/bandwidth_test.js]: pass this on the constructor too.

File: src/js/bandwidth_test.js [hunk 2]
```diff
@@ -227,75 +227,95 @@ VideoBandwidthTest.prototype = {
...
+  this.testDurationMs = 5 * 60 * 1000;
+  this.sendIntervalMs = 100;
+  this.delays = [];
+  this.recvTimeStamps = [];
+  this.running = false;
+}
+
+WiFiPeriodicScanTest.prototype = {
+  run: function(candidateFilter) {
+    var start = function(candidateFilter, config) {
```
Comment (andresusanopinto) [inline src/js/bandwidth_test.js]: make this a prototype function and just do:

Call.asyncCreateTurnConfig(this.start.bind(this), ...);

File: src/js/bandwidth_test.js (hunk 1 above)
Comment (KaptenJansson) [inline src/js/bandwidth_test.js]: Done

**Comment by andresusanopinto** [comment]
lgtm

**Pull request merged**